AchieveIt.API/Models/RegisterStudentModel.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System;
+
+namespace AchieveIt.API.Model
+{
+    public class RegisterStudentModel
```

Comment:
Looks like RegisterTeacherModel class is duplicated with RegisterStudentModel. Maybe we should use inheritance here?